pkg/kubelet/dockertools/manager.go

```diff
@@ -1156,6 +1156,7 @@ func (dm *DockerManager) killContainer(containerID types.UID) error {
 	if inspect != nil && inspect.Config != nil && inspect.Config.Labels != nil {
 		preStop, found = inspect.Config.Labels[kubernetesPodLabel]
 	}
+	gracePeriod := uint(30)
```

Comment:
Why is this defaulting in kubelet rather than in the API defaulting code? I'd prefer the latter.